tests/slurm_plugin/test_clustermgtd.py

```diff
@@ -595,25 +593,36 @@ def test_fail_scheduled_events_health_check(instance_health_state, expected_resu
         "expected_failed_nodes",
         "current_node_in_replacement",
         "expected_node_in_replacement",
+        "expected_bootstrap_failure_node",
     ),
     [
         (
             ClusterManager.HealthCheckTypes.scheduled_event,
-            [True, False],
-            [False, True],
-            ["queue1-st-c5xlarge-2"],
+            [True, False, True, False],
+            [False, True, False, True],
+            ["queue1-st-c5xlarge-2", "queue1-st-c5xlarge-4"],
             {"some_node_in_replacement1"},
             {"some_node_in_replacement1"},
+            SlurmNode("queue1-st-c5xlarge-2", "ip-2", "host-2", "ALLOCATED#+CLOUD", "queue1"),
         ),
         (
             ClusterManager.HealthCheckTypes.ec2_health,
-            [True, False],
-            [False, True],
-            ["queue1-st-c5xlarge-1"],
-            {"some_node_in_replacement1", "queue1-st-c5xlarge-1"},
```

Comment:
Are we reducing test coverage here? We were supposed to test the case that if a node in instance replacement fails health check, we take it out of node in replacement so it will be replaced later.

Reply:
Right! I didn't realize that, add it back, also add the test for case static node fail bootstrap